src/TransactionHandler.ts

```diff
@@ -0,0 +1,28 @@
+import ContractEntityHandler from './handlers/ContractEntityHandler';
+import { HandlerNotFoundException } from './Exception';
+
+export default class TransactionHandler {
+  private readonly handlers: Record<string, ContractEntityHandler<any>>;
+
+  public constructor(handlers: Record<string, ContractEntityHandler<any>>) {
+    this.handlers = handlers;
+    this.handle = this.handle.bind(this);
+  }
+
+  /**
+   * This method accept bulkTransactions and handles them with specific handlers.
+   * @param bulkTransactions List of bulkTransactions.
+   */
+  public handle(bulkTransactions: any): void {
```

Comment:
Since subscription code is merged, the method can be integrated with GraphClient.subscribe method 🏗 🏛